.travis.yml

```diff
@@ -0,0 +1,10 @@
+dist: trusty
```

Comment:
Needed?

Reply:
Would be helpful to run on multiple systems to check portability, but if it can be omitted, it's a good base case.